### DISCUSSION TITLE
What is the AI feature?

### DISCUSSION THREAD

**Discussion by gerroon** [wallabag/ios-app#411] [Q&A]
Hi

I see that there is a new AI thing in the release note in the app store? What is it?

**Comment by bourvill** [comment]
Hello @gerroon 

The new AI feature provides a smart assistant to create synthesis or suggest tag for you entries at this time. But more features can be added in future, any suggestions are welcome.

This features are powered by chatGPT and it requires to subscribe to wallabag Plus plan inside the app. The ai feature is only available on Apple device.

When you want a synthesis or tag suggestion, only your entry is sended and context is added in our server side ( the code will be open sourced soon ) 

I hope this help you

**Reply by gerroon** [reply]
That makes sense, thanks.
